warehouse/models/rt_views/gtfs_rt_vs_sched_routes.sql

```diff
@@ -0,0 +1,69 @@
+WITH 
+-- selecting the distinct trips from GTFS Vehicle Postions for two operators (SamTrans 290 and Big Blue Bus 300) and two months
+vp_trips AS (
+    SELECT DISTINCT
+     calitp_itp_id,
+     calitp_url_number,
+     date AS service_date,
+     trip_id AS vp_trip_id,
+     trip_route_id,
```

Comment:
Just want to note that technically both `trip_id` and `route_id` are [only conditionally required](https://gtfs.org/realtime/reference/#message-vehicleposition) in vehicle positions based on kind of complicated logic... If we were to expand this table to cover all operators, we may need to add more logic to handle cases where `trip_id` is null. (This is something that we could just note in a comment here -- for example, could add a comment linking to the docs and noting that changes may be needed when generalizing.)

Also, it doesn't look like you use `route_id` from this table later, so you could possibly drop it from the `SELECT` statement here, although if we were to implement the more complicated conditional logic later we would need it then.

Reply:
Made a note of it and removed route_id for now!